examples/ZMQPP/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-ZeroMQ)
+
+hunter_add_package(ZeroMQ)
+hunter_add_package(ZMQPP)
+
+find_package(ZMQPP)
```

Comment:
It has been fixed. But Huston I have a problem. The cache is missing in ZeroMQ too. Can I commit fix here, or do I create a new pull request.